### ISSUE TITLE
[nfs-server-provisioner] Update nfs-provisioner image's tag

### ISSUE THREAD

**Issue by roonie007** [helm/charts#18965]
**Is your feature request related to a problem? Yes.**

Can you upgrade the helm so we can get benefits of the changes of this PR https://github.com/kubernetes-incubator/external-storage/pull/1214

**Describe the solution you'd like**
Make the needed changes to the Helm package

**Comment by lurraca** [comment]
+1. Is this something being looked at? or should I try to make this contribution?

**Comment by roonie007** [comment]
@lurraca  Go ahead :+1:

**Comment by cedricve** [comment]
any chance this get included? really need this..

**Comment by cedricve** [comment]
@lurraca is this still something on your todo list or do you need help?

**Comment by cedricve** [comment]
Expanding the arguments list here, would do the trick I believe.
https://github.com/helm/charts/blob/master/stable/nfs-server-provisioner/templates/statefulset.yaml#L52

**Comment by cedricve** [comment]
sorry to disturb @naseemkullah, but you have any thoughts of implementing this fix. I think a lot of people would benefit from this. (see kubernetes-incubator/external-storage#1214).a

the problem is that NFS is not useable when a the pod gets rescheduled to a different node. Because all connnected nfs clients will receive a staled file handle.

**Comment by naseemkullah** [comment]
> When running a cloud based kubernetes service (like Googles GKE service) set this to `false` as it might affect client connections on restarts of the nfs provisioner pod. Default true.

Sounds good to me! Want to open a PR to add the flag?

**Comment by naseemkullah** [comment]
Better yet, we should add a value to add any additional args we want to. something like `extraArgs:` you can find various examples in the stable repo of charts that have this.

**Comment by naseemkullah** [comment]
We could also instead of `extraArgs` simply have `args` and move `- "-provisioner={{ template "nfs-provisioner.provisionerName" . }}"` to the values file.

**Comment by cedricve** [comment]
@naseemkullah can you check my PR?

**Comment by cedricve** [comment]
> We could also instead of `extraArgs` simply have `args` and move `- "-provisioner={{ template "nfs-provisioner.provisionerName" . }}"` to the values file.

Sorry I missed your last comment. Not sure how to achieve that?

**Comment by naseemkullah** [comment]
No worries, please see https://github.com/helm/charts/pull/20371 , with this you will be able to add that additional command line param, but we may follow up with yet another PR to have it on by default if it is really required/recommended when using in k8s

**Comment by cedricve** [comment]
For k8s it has to be on from my understanding. But @thirdeyenick from following the original issue (https://github.com/kubernetes-incubator/external-storage/pull/1214) will have a better explanation; although already very good explained.

**Comment by lurraca** [comment]
This is really good work. Thanks @cedricve and @naseemkullah. Now we are waiting on the image to be cut from the nfs-provisioner 2.3.0 to be pushed to to quay.io.

**Comment by cedricve** [comment]
thanks @lurraca if it would be it was already ready 5months ago, but it's out of our control. We can only increase the urge on your existing Github issue. For the time being, I simply compiled it to my private registry.
